c7n/resources/quicksight.py

```diff
@@ -34,6 +34,25 @@ class resource_type(query.TypeInfo):
     }
 
 
+@QuicksightUser.action_registry.register('delete')
+class DeleteUserAction(BaseAction):
+    schema = type_schema('delete',)
+    permissions = ('quicksight:DeleteUser',)
+
+    def process(self, resources):
+        session = local_session(self.manager.session_factory)
+        client = session.client(self.manager.resource_type.service)
+        account_id = self.manager.config.account_id
+        for r in resources:
+            self.manager.retry(
+                client.delete_user,
+                AwsAccountId=account_id,
+                Namespace='default',
```

Comment:
I had a moment here thinking "is it OK to hardcode this?" but [the docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/quicksight/client/delete_user.html) make that pretty clear :sweat_smile: 

> Namespace (string) –
> [REQUIRED]
> The namespace. Currently, you should set this to default.